tests/specs/test-files/Logo.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="378" height="171" viewBox="0 0 378 171" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
are you sure that you need this image
change the name to cebab case

Reply:
this file is required for upload testing

Reply:
or i can use any else .ts file

Reply:
now the dashboard.ts file is used for testing uploads)